cmd/sync/sync.go

```diff
@@ -109,6 +123,23 @@ func main() {
 				message.AccessionID,
 				message.DecryptedChecksums)
 
+			// Get the header from db
+			header, err := db.GetHeaderStableId(message.AccessionID)
+			if err != nil {
+				log.Errorf("GetHeaderStableId failed "+
```

Comment:
There are a few error messages that are very similar to this. Might be worth looking into a helper function to format the repeating parts!

Reply:
We discussed with Alex about error messages and nacks and we will add it probably for refactoring for all the services.